docs/docs/sample-scripts/teams/list-teams-decluttering/index.mdx

```diff
@@ -0,0 +1,150 @@
+---
+tags:
+  - decluttering
+  - insights
+  - usage
+  - governance
+  - teams
+---
+
+# Lists information on decluttering teams
+
+Author: [Tobias Maestrini](https://github.com/tmaestrini)
+
+Get insights about the teams in your tenant. Please keep in mind that you see only the teams that you have access to. 
+For a full overview and a complete decluttering experience, the calling user needs to have admin rights in SPO and Teams.
+
+=== "PowerShell"
```

Comment:
Hi @tmaestrini 

Quick comment from my side. Noticed that you used our old docs syntax. Could have a look at another sample script and use the same way of working (with `Tabs` and `TabItem` tags)? Thanks!

Reply:
@milanholemans Done! :)